src/ChaiWrapper.js

```diff
@@ -104,15 +104,23 @@ export default class ChaiWrapper {
         return _super.apply(this, arguments)
       }
 
-      assertion.call(this, {
+      const config = {
         markup: () => debug(wrapper),
         sig: inspect(wrapper),
         wrapper,
         arg1,
-        arg2,
         flag,
         inspect
-      })
+      }
+
+      /**
+       * See comment below about arg2 use.
```

Comment:
@ayrton [this comment](https://github.com/wbyoung/chai-enzyme/blob/8610873a9d1e9804fce705455c8f04f89107bad2/src/ChaiWrapper.js#L172).